### PR TITLE
K3S and Helm version upgraded

### PR DESCRIPTION
**Pull request by TheYkk**
## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?
All upgrades include minor change. Mostly bug fixes.Upgrades does not require any special change.
Edit: I tested with simple k3d cluster and run tihs `go run main.go install kubernetes-dashboard`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by TheYkk** [comment]
@Waterdrips

File: pkg/config/config.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 )
 
 // K3sVersion default version
-const K3sVersion = "v1.17.2+k3s1"
+const K3sVersion = "v1.17.3+k3s1"
```
[the remark this answers is not in the document]
Comment (TheYkk) [reply pkg/config/config.go]: i did.

**Comment by TheYkk** [comment]
k8s dashboard works fine
image

**Comment by Waterdrips** [comment]
Thats great.

Can you use the following to test, so we are sure the binary is generated correctly:

`go build && ./arkade install kubernetes-dashboard`

And then we need to validate that the correct helm version is installed to the correct place, so you would need to do something like:

```sh
rm -rf ~/.arkade 
go build && ./arkade install openfaas
# <output should include installing your new helm version>

# then
 ~/.arkade/bin/helm3/helm version
# <output should show your new version>
```

This is because the kubernetes-dashboard doesn't use helm to install and therefore no testing has been done for this new version.

**Comment by TheYkk** [comment]
Go build is fine
image
And Helm is fine
image
image

**Comment by alexellis** [comment]
Thank you @TheYkk

**Pull request merged**